js/okcoin.js

```diff
@@ -2575,6 +2586,93 @@ module.exports = class okcoin extends Exchange {
         return address;
     }
 
+    async transfer (code, amount, fromAccount, toAccount, params = {}) {
+        await this.loadMarkets ();
+        const currency = this.currency (code);
+        const accountsByType = this.safeValue (this.options, 'accountsByType', {});
+        const fromId = this.safeString (accountsByType, fromAccount, fromAccount);
+        const toId = this.safeString (accountsByType, toAccount, toAccount);
+        const request = {
+            'amount': this.currencyToPrecision (code, amount),
+            'currency': currency['id'],
+            'from': fromId,
+            'to': toId,
+            'type': '0',
+        };
+        if (fromId === 'main') {
+            request['type'] = 1;
```

Comment:
According to OKCoin's API docs `type` must be a string: https://www.okcoin.com/docs/en/#account-transfer